client/app/bundles/momentCard/components/MomentCardCategories.jsx

```diff
@@ -0,0 +1,25 @@
+// @flow
+import React from 'react';
+import css from './MomentCard.scss';
+
+type MomentCardCategoriesState = {};
+
+type MomentCardCategoriesProp = {
+  category: any;
+};
+
+export default class MomentCardCategories
+  extends React.Component<MomentCardCategoriesProp, MomentCardCategoriesState> {
```

Comment:
You can omit MomentCardViewersState as the second parameter (it's optional).

On the following lines, you can omit `props: MomentCardViewersProp` and `state: MomentCardViewersState` because they should already be defined as part of React.Component< MomentCardViewersProp>.